Resources/Prototypes/_DV/Entities/Structures/Machines/autoclave.yml

```diff
@@ -0,0 +1,75 @@
+- type: entity
+  parent: [ BaseMachinePowered, SmallConstructibleMachine ]
+  id: Autoclave
+  name: autoclave
+  description: Makes anything you put in it very clean
+  components:
+  - type: Appearance
+  - type: Autoclave
+  - type: SurgeryCleansDirt
+    dirtAmount: 10
+    dnaAmount: 2
+  - type: Sprite
+    sprite: _DV/Structures/Machines/autoclave.rsi
+    snapCardinals: true
+    layers:
+    - state: base
+      map: ["enum.StorageVisualLayers.Base"]
+    - state: door_closed
+      map: ["enum.StorageVisualLayers.Door"]
+    - state: light_active
+      shader: unshaded
+      visible: false
+      map: ["enum.AutoclaveVisuals.ProcessingLayer"]
+    - state: light_inactive
+      shader: unshaded
+      visible: false
+      map: ["enum.AutoclaveVisuals.IdleLayer"]
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 100
+      behaviors:
+      - !type:DoActsBehavior
+        acts: ["Breakage"]
```

Comment:
remove this override so it's not silent, the default is the same with 200 threshold to not play sound during nuke